### PR TITLE
Logging config mainnet

### PR DESCRIPTION
**Pull request by CodiePP**
adapting configuration for 'mainnet.sh' script.

### CODE REVIEW

File: benchmarking/chain-sync/configuration/log-config-byron-proxy.yaml [hunk 1]
```diff
@@ -18,6 +18,9 @@ hasEKG: 12789
 
 # if wanted, the Prometheus interface is using this port:
 hasPrometheus: 12799
+#hasPrometheus: 
+#  - "localhost"
+#  - 12799
```
Comment (mrBliss) [inline benchmarking/chain-sync/configuration/log-config-byron-proxy.yaml]: I don't like seeing things comment out.

File: benchmarking/chain-sync/launch_mainnet_proxy.sh [hunk 2]
```diff
@@ -11,7 +11,7 @@ set -euo pipefail
 
 cd ${TARGETDIR}
 
-cabal new-run exe:cardano-byron-proxy -- \
+cabal v2-run exe:cardano-byron-proxy -- \
```
Comment (mrBliss) [inline benchmarking/chain-sync/launch_mainnet_proxy.sh]: Shouldn't this change have been part of #384? How many more changes like this did #384 miss?

File: configuration/configuration-mainnet.yaml [hunk 3]
```diff
@@ -26,15 +26,15 @@ defaultBackends:
...
     scFormat: ScText
-    scRotation: null
 
 # if not indicated otherwise, then log output is directed to this:
 defaultScribes:
-  - - StdoutSK
-    - stdout
+  - - FileSK
+    - "logs/mainnet.log"
+    #  - - DevNullSK
```
Comment (mrBliss) [inline configuration/configuration-mainnet.yaml]: Commented out

File: configuration/configuration-mainnet.yaml [hunk 4]
```diff
@@ -88,7 +100,8 @@ NumCoreNodes: 1
 RequiresNetworkMagic: RequiresNoMagic
 PBftSignatureThreshold: 0.7
 TurnOnLogging: True
-ViewMode: SimpleView
+#ViewMode: SimpleView
```
Comment (mrBliss) [inline configuration/configuration-mainnet.yaml]: Commented out

File: scripts/mainnet.sh [hunk 5]
```diff
@@ -11,8 +11,8 @@ ARGS=(
         --genesis-hash            "5f20df933584822601f9e3f8c024eb5eb252fe8cefb24d1317dc3d432e940ebb"
         --topology                "${TOPOLOGY}"
         --socket-dir              "./socket/"
-        --config                  "./configuration/mainnet.yaml"
+        --config                  "./configuration/configuration-mainnet.yaml"
         --port                    7776
 )
 
-${RUNNER} exe:cardano-node "${ARGS[@]}"
+${RUNNER} exe:cardano-node "${ARGS[@]}" $*
```
Comment (mrBliss) [inline scripts/mainnet.sh]: :+1: Is this now part of all scripts that run the node?

Comment (CodiePP) [reply scripts/mainnet.sh]: no just this one to pass in +RTS flags.

File: configuration/configuration-mainnet.yaml (hunk 4 above)
Comment (CodiePP) [reply configuration/configuration-mainnet.yaml]: allows changing between modes.

File: configuration/configuration-mainnet.yaml (hunk 3 above)
Comment (CodiePP) [reply configuration/configuration-mainnet.yaml]: clearly a leftover that should have been removed.

File: benchmarking/chain-sync/configuration/log-config-byron-proxy.yaml (hunk 1 above)
Comment (CodiePP) [reply benchmarking/chain-sync/configuration/log-config-byron-proxy.yaml]: This is in preparation. The byron-proxy has not updated its dependencies yet.

**Comment by CodiePP** [comment]
obsolete, replaced by #408

**Pull request closed without merging**